.gitignore

```diff
@@ -19,6 +19,9 @@ CorsixTH/config.txt
 *.exe
 Thumbs.db
 
+# For built binary
+CorsixTH/CorsixTH
```

Comment:
I generally discourage building in the root source directory. Building out of source isn't well supported yet, but building in a build*/ subdirectory works really well.